docs/quality.md

```diff
@@ -1,3 +1,60 @@
 # Quality
 
-> @wip
+- Don't want to break this for driver, mongoose
+- Want input from these maintainers on establishing best practies, not to just
+  create more work
+- Result of this will be incorporated into our tooling,
+  [mj](https://github.com/rueckstiess/mj), for automated organization wide
+  checking and scaffolding for new projects
+
+## Testing
+
+- `npm test` returns a non-zero exit code
+- travisci hook
+
+## Code Coverage
+
+- do it if possible
+- tooling will template out istanbul + mocha + coveralls.io
+
+## Documentation
+
+- A lot of different needs here
+- For general small modules, examples in README + comments where it makes sense
+- If you expose more than `module.exports = function(){}`, use jsdoc tags + `dox --api`
+- Tooling will publish a pretty version to github pages for you
+
+## Style
+
+- A `.jshintrc` [example](https://github.com/imlucas/khaos-amp/blob/master/template/.jshintrc)
+- A `.jsfmtrc` [example](https://github.com/imlucas/khaos-amp/blob/master/template/.jsfmtrc)
```

Comment:
Nowadays I prefer using [jsrc and specifying style rules in package.json](https://github.com/vkarpov15/kareem/blob/master/package.json#L23-27), so `npm install` ensures that you get the style rules. Makes it easier for contributors because they get a copy of the style guide without having to overwrite any dotfiles they may already have.